### DISCUSSION TITLE
Is it possible to track dynamically created pods?

### DISCUSSION THREAD

**Discussion by jonathanasdf** [argoproj/argo-workflows#8461] [Q&A]
I'm trying to run a Spark job inside an Argo workflow, and the spark job can create additional executor pods dynamically. Is there a way to track those pods so that they show up under the argo watch command?

**Comment by terrytangyuan** [comment]
Not a solution to show those pods in watch command but you can add this label “workflows.argoproj.io/workflow” if you want to retrieve the logs.

An example: https://github.com/argoproj/argo-workflows/blob/master/examples/k8s-resource-log-selector.yaml
